safekeeper/src/metrics.rs

```diff
@@ -204,6 +204,28 @@ pub static WAL_BACKUP_TASKS: Lazy<IntCounterPair> = Lazy::new(|| {
     )
     .expect("Failed to register safekeeper_wal_backup_tasks_finished_total counter")
 });
+pub static WAL_RECEIVERS: Lazy<IntGauge> = Lazy::new(|| {
+    register_int_gauge!(
+        "safekeeper_wal_receivers",
```

Comment:
We can also get this from summing up `safekeeper_connected_computes` across all timelines, but seemed useful to have a total gauge. Particularly for comparison with the number of samples from `safekeeper_wal_receiver_queue_depth`.